src/domain/safe/entities/schemas/erc721-transfer.schema.ts

```diff
@@ -1,5 +1,21 @@
 import { Schema } from 'ajv';
+import { z } from 'zod';
+import { AddressSchema } from '@/validation/entities/schemas/address.schema';
+import { HexSchema } from '@/validation/entities/schemas/hex.schema';
 
+export const Erc721TransferSchema = z.object({
+  type: z.literal('ERC721_TRANSFER'),
+  executionDate: z.coerce.date(),
+  blockNumber: z.number(),
+  transactionHash: HexSchema,
+  to: AddressSchema,
+  from: AddressSchema,
+  tokenId: z.string(),
+  tokenAddress: AddressSchema.nullish().default(null),
```

Comment:
The same should apply to the `tokenAddress` of an ERC721 transfer.

Reply:
Fixed in a4fb3f8ccb411e2af223a70d76f90c5e0fce198b.